src/bincode/bincode.zig

```diff
@@ -797,6 +808,21 @@ const ShredType = enum(u8) {
     pub const @"!bincode-config" = ShredTypeConfig();
 };
 
+pub fn testRoundTrip(comptime T: type, test_data: anytype) !void {
```

Comment:
I think some documentation briefly explaining this method and the constraints on test_data would be useful

Reply:
87ae4a59b84503a0adc38e2b59ccd0fbe2f59322